packages/contractkit/src/kit.ts

```diff
@@ -75,6 +76,18 @@ export class ContractKit {
     this.contracts = new WrapperCache(this)
   }
 
+  async getTotalBalance(address: string): Promise<BigNumber> {
```

Comment:
What about returning an object with all the different type of balances?

Something like:
```ts
interface AccountBalance {
   gold: BigNumber,
   usd: BigNumber.
   lockedGold: BigNumber
}
```
?

Also, converting "stable -> gold" sounds like a good add to the `Exchange` wrapper.. probably already has one, since you can ask a quote of how many gold will i received when exchanging XXX stable. In fact, it's call `Exchange.quoteUsdSell()`